sdk/repository.go

```diff
@@ -43,16 +43,17 @@ type ProjectRepositoryAnalysis struct {
 }
 
 type ProjectRepositoryData struct {
-	HookEventUUID   string                        `json:"hook_event_uuid"`
-	HookEventKey    string                        `json:"hook_event_key"`
-	OperationUUID   string                        `json:"operation_uuid"`
-	CommitCheck     bool                          `json:"commit_check"`
-	SignKeyID       string                        `json:"sign_key_id"`
-	CDSUserName     string                        `json:"cds_username"`
-	CDSUserID       string                        `json:"cds_username_id"`
-	CDSAdminWithMFA bool                          `json:"cds_admin_mfa"`
-	Error           string                        `json:"error"`
-	Entities        []ProjectRepositoryDataEntity `json:"entities"`
+	HookEventUUID             string                        `json:"hook_event_uuid"`
+	HookEventKey              string                        `json:"hook_event_key"`
+	OperationUUID             string                        `json:"operation_uuid"`
+	CommitCheck               bool                          `json:"commit_check"`
+	SignKeyID                 string                        `json:"sign_key_id"`
+	DeprecatedCDSUserName     string                        `json:"cds_username"`    // Deprecated
+	DeprecatedCDSUserID       string                        `json:"cds_username_id"` // Deprecated
+	DeprecatedCDSAdminWithMFA bool                          `json:"cds_admin_mfa"`   // Deprecated
+	Error                     string                        `json:"error"`
+	Entities                  []ProjectRepositoryDataEntity `json:"entities"`
+	Initiator                 *V2Initiator                  `json:"initiator"`
```

Comment:
Update UI to display user from Initiator if it exists

Reply:
compat code on API side done